src/dump.c

```diff
@@ -2470,7 +2543,7 @@ static void jl_insert_backedges(jl_array_t *list, jl_array_t *targets)
                 int32_t idx = idxs[j];
                 jl_value_t *callee = jl_array_ptr_ref(targets, idx * 2);
                 if (jl_is_method_instance(callee)) {
-                    jl_method_instance_add_backedge((jl_method_instance_t*)callee, caller);
+                    jl_method_instance_add_backedge((jl_method_instance_t*)callee, NULL, caller);
```

Comment:
This should also need the `invoke`-types for the backedge too (sometimes)?

Reply:
Ah, I guess we need to expand `targets` to include the invokesig?